data_analysis/plots.py

```diff
@@ -1213,3 +1218,185 @@ def make_descriptive_plots(
         f"{start_date}_to_{ridership_end_date}"
         f"_all_day_types",
     )
+
+
+def calculate_ratio_per_ward(
+    ward_df: pd.DataFrame, summary_df: pd.DataFrame
+) -> pd.DataFrame:
+    """
+    Args:
+        ward_df (pd.DataFrame): DataFrame containing all routes
```

Comment:
Would it be possible to specify the schema of this df (what are the important columns & what do they mean)?